### DISCUSSION TITLE
Right-click on spelling mistake fills full screen

### DISCUSSION THREAD

**Discussion by mal-tee** [texstudio-org/texstudio#1794] [Q&A]
Hi!

If I right-click on some underlined words, the resulting menu fills the whole screen and is therefore unusable. Is there an option to limit the amount of suggested words?

I am running `TeXstudio 3.0.4 (Build: 3.0.4+ds-1) Using Qt Version 5.15.2, compiled with Qt 5.15.2 R` with Language-Tool stable with `en` ngrams and word2vec.

![image](https://user-images.githubusercontent.com/45886910/135836591-13f771fe-f354-4bea-bef5-a0146d1e9902.png)
![image](https://user-images.githubusercontent.com/45886910/135836632-d8892025-be22-4f1a-818f-4e372e4a1ecd.png)

Thanks in advance!

**Comment by sunderme** [comment]
there is none in txs. Maybe in LT ?
